### ISSUE TITLE
some process is not stoppable

### ISSUE THREAD

**Issue by mave99a** [ArcBlock/forge-cli#147]
Even stop --all doesn't help
```
$ forge stop --all
──────────────
✔ Current Chain: chain-1
──────────────
⚠ chain-1 is not started!
```

Some process is still there: 
```
maos-MacBook-Pro:forge-python-starter mao$ forge ps
──────────────
✔ Current Chain: chain-1
──────────────

 Chain: chain-1
┌───────────────┬──────────┬──────────┬───────────────┬────────────────────┐
│ Name          │ PID      │ Uptime   │ Memory        │ CPU                │
├───────────────┼──────────┼──────────┼───────────────┼────────────────────┤
│ web           │ 66955    │ 3d       │ 4.8 MB        │ 0.01 %             │
└───────────────┴──────────┴──────────┴───────────────┴────────────────────┘

maos-MacBook-Pro:forge-python-starter mao$ forge version
──────────────
✔ Current Chain: chain-1
──────────────
forge-core version 0.36.4 on darwin
forge-cli version 0.36.0
forge_web 0.36.4
simulator 0.36.4
forge_workshop 0.36.4
```

**Comment by polunzh** [comment]
@mave99a 
This bug was fixed at lastest version `0.36.4`, try to upgrade to latest.

**Comment by polunzh** [comment]
```
$ forge ps                                                                                                                                                 │-rw-r--r--   1 zhenqiang  staff   2.3K Jul  9 09:11 CODE_OF_CONDUCT.md
──────────────                                                                                                                                             │-rw-r--r--   1 zhenqiang  staff   8.0K Jul  9 09:11 CONTRIBUTING.md
✔ Current Chain: chain-1                                                                                                                                   │-rw-r--r--   1 zhenqiang  staff    34K Jul  9 09:11 COPYING.txt
──────────────                                                                                                                                             │-rw-r--r--   1 zhenqiang  staff   146K Jul  9 09:11 README.md
                                                                                                                                                           │drwxr-xr-x   3 zhenqiang  staff    96B Jul  9 09:11 autoload
 Chain: chain-1                                                                                                                                            │drwxr-xr-x   7 zhenqiang  staff   224B Jul  9 09:11 azure
┌───────────────┬──────────┬──────────┬───────────────┬────────────────────┐                                                                               │-rw-r--r--   1 zhenqiang  staff   3.0K Jul  9 09:11 azure-pipelines.yml
│ Name          │ PID      │ Uptime   │ Memory        │ CPU                │                                                                               │-rw-r--r--   1 zhenqiang  staff   432B Jul  9 09:11 codecov.yml
├───────────────┼──────────┼──────────┼───────────────┼────────────────────┤                                                                               │drwxr-xr-x   4 zhenqiang  staff   128B Jul  9 09:45 doc
│ web           │ 9306     │ 32s      │ 99.4 MB       │ 9.78 %             │                                                                               │-rwxr-xr-x   1 zhenqiang  staff   1.4K Jul  9 09:11 install.py
└───────────────┴──────────┴──────────┴───────────────┴────────────────────┘                                                                               │-rwxr-xr-x   1 zhenqiang  staff   328B Jul  9 09:11 install.sh
                                                                                                                                                           │drwxr-xr-x   3 zhenqiang  staff    96B Jul  9 09:11 plugin
                                                                                                                                                           │-rwxr-xr-x   1 zhenqiang  staff   153B Jul  9 09:11 print_todos.sh
# zhenqiang @ zhenqiangMacBook in ~/workcode/forge-cli on git:master o [10:06:31]                                                                          │drwxr-xr-x   4 zhenqiang  staff   128B Jul  9 09:11 python
$ forge stop                                                                                                                                               │-rwxr-xr-x   1 zhenqiang  staff   3.4K Jul  9 09:11 run_tests.py
──────────────                                                                                                                                             │drwxr-xr-x   7 zhenqiang  staff   224B Jul  9 09:11 third_party
✔ Current Chain: chain-1                                                                                                                                   │-rw-r--r--   1 zhenqiang  staff   174B Jul  9 09:11 tox.ini
──────────────                                                                                                                                             │
ℹ Stopping chain-1 chain...                                                                                                                                │# zhenqiang @ zhenqiangMacBook in ~/.vim/bundle/YouCompleteMe on git:master x [9:57:16]
✔ Chain chain-1 stopped!                                                                                                                                   │$ git submodule update --init --recursive
                                                                                                                                                           │Cloning into '/Users/zhenqiang/.vim/bundle/YouCompleteMe/third_party/requests_deps/certifi'...
# zhenqiang @ zhenqiangMacBook in ~/workcode/forge-cli on git:master o [10:06:45]                                                                          │Cloning into '/Users/zhenqiang/.vim/bundle/YouCompleteMe/third_party/requests_deps/chardet'...
$ forge ps                                                                                                                                                 │
ℹ forge daemon not started yet, start with forge start                                                                                                     │
                                                                                                                                                           │
# zhenqiang @ zhenqiangMacBook in ~/workcode/forge-cli on git:master o [10:06:50]                                                                          │Cloning into '/Users/zhenqiang/.vim/bundle/YouCompleteMe/third_party/requests_deps/idna'...
$ ps -ef | grep forge                                                                                                                                      │Cloning into '/Users/zhenqiang/.vim/bundle/YouCompleteMe/third_party/requests_deps/requests'...
  501  9682  3501   0 10:06AM ttys001    0:00.00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=│
.svn forge
```

Version:
```
$ forge version                                                                                                                                            │
forge-core version 0.36.4 on darwin                                                                                                                        │
forge-cli version 0.36.4                                                                                                                                   │Cloning into '/Users/zhenqiang/.vim/bundle/YouCompleteMe/third_party/requests_deps/idna'...
forge_web 0.36.4                                                                                                                                           │Cloning into '/Users/zhenqiang/.vim/bundle/YouCompleteMe/third_party/requests_deps/requests'...
simulator 0.36.4                                                                                                                                           │
forge_workshop 0.36.4
```
